docs/filemanagement.rst

```diff
@@ -208,6 +229,54 @@ like so:
 
     content_string = file4.GetContentString(encoding='ISO-8859-1')
 
+Upload data as bytes in memory buffer
+--------------------------------------
+
+Data can be kept as bytes in an in-memory buffer when we use the io module’s
```

Comment:
io -> `io`

Reply:
could you be more specific? i dont understand

Reply:
sorry, it's stylistic change, make certain tech terms, var names, etc code inlined

text io text 

vs

text `io` text

Reply:
do you want me to highlight the word io? like in bold or italic?

Reply:
I mean `inline literal` from this doc I think - https://docutils.sourceforge.io/docs/user/rst/quickref.html#inline-markup

Reply:
ahh i got it now, sorry i didnt know reST have inline literals